### PR TITLE
Refactor

### LINKED ISSUE #78: Rename nodes, classes etc
We should select a standard for all names. It is quite chaotic now. 

I will add more info soon

### LINKED ISSUE #91: Unused Files
There are lots of leftover files which have become obsolete. These should be removed to prevent confusion from those not in the know.

### PR DESCRIPTION
**Pull request by Rezenders**
Refactoring to resolve #91, resolve #78 and partially #95 but not enough.
I still didn't test the PR, so I will keep it as a draft for now.
But in the mean time, can you (@jeroenzwan @EGAlberts) check if you agree on the files I removed, the renaming of the pkgs and nodes, and the current organization of the files?

I will change the repo name after this PR is merged.

### CODE REVIEW

File: suave/launch/sauve.launch.py [hunk 1]
```diff
@@ -67,32 +54,42 @@ def generate_launch_description():
...
+        package='suave',
+        executable='spiral_search',
         output='screen'
     )
 
-    follow_pipeline_lc = Node(
-        package='pipeline_inspection_metacontrol',
-        executable='follow_pipeline_lc',
+    follow_pipeline_node = Node(
+        package='suave_metacontrol',
```
Comment (EGAlberts) [inline suave/launch/sauve.launch.py:70]: should line 70 still say suave_metacontrol?

**Comment by EGAlberts** [comment]
It looks good to me. I'm eager to update my branches to reflect the refactoring :)

File: suave/launch/sauve.launch.py (hunk 1 above)
Comment (Rezenders) [reply suave/launch/sauve.launch.py:70]: No, it is wrong. Thanks for pointing out.

File: suave/launch/sauve.launch.py [hunk 2]
```diff
@@ -12,19 +12,6 @@
```
Comment (EGAlberts) [inline suave/launch/sauve.launch.py:12]: Also this filename has a typo

**Pull request merged**